Bottom: 9cca0d7b8efd4351af12572edb06d08fab4dfc3a
Top:    4eb1f2e4bd04687ac4e603de337e8c2633a7d293
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 02:41:34 +0100

Refresh of add-soldermask-layers

---

diff --git a/src/global.h b/src/global.h
index 41f4988..d168a46 100644
--- a/src/global.h
+++ b/src/global.h
@@ -397,11 +397,7 @@ typedef struct			/* holds all objects */
   rtree_t *via_tree, *element_tree, *pin_tree, *pad_tree, *name_tree[3],	/* for element names */
    *rat_tree;
   struct PCBType *pcb;
-<<<<<<< current
   LayerType Layer[MAX_LAYER + EXTRA_LAYERS];
-=======
-  LayerType Layer[MAX_LAYER + EXTRA_LAYERS];	/* add 2 silkscreen layers and 2 soldermask layers */
->>>>>>> patched
   int polyClip;
 } DataType;
